src/Markdig/Extensions/DefinitionLists/DefinitionListParser.cs

```diff
@@ -103,6 +102,19 @@ public override BlockState TryOpen(BlockProcessor processor)
             return BlockState.Continue;
         }
 
+        private static DefinitionList GetCurrentDefinitionList(ParagraphBlock paragraphBlock, ContainerBlock previousParent)
+        {
+            var index = previousParent.IndexOf(paragraphBlock) - 1;
+            if (index < 0) return null;
+            var lastBlock = previousParent[index];
+            if (lastBlock is BlankLineBlock)
```

Comment:
Don't remember exactly if we can have multiple Blanklines, better to double check.